### ISSUE TITLE
Auto start on boot?

### ISSUE THREAD

**Issue by Smiggel** [janoside/btc-rpc-explorer#60]
Is it possible to run the explorer on boot and if so how?

I would really like to have it run when I boot my system. I run the node on a Pi with Raspbian, Bitcoin Core 0.16.2 and Lightning 0.4.2.

**Comment by dawiepoolman** [comment]
I am sure we can write a .bat file with the arguments to boot the server.
I will test as soon as I get my connection to work.  We seem to have the exact same node setup.

**Comment by dawiepoolman** [comment]
yep, pretty easy.  just put this in a bat file and put the bat file in startup:

cd C:\Users\YourUsername\Documents\btc-rpc-explorer
npm start
